sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/StorageBlobAsyncClient.java

```diff
@@ -374,6 +376,73 @@ public void getBlobProperties(String containerName,
             callback);
     }
 
+    /**
+     * Sets the blob's tier.
+     *
+     * @param containerName The container name.
+     * @param blobName      The blob name.
+     * @param tier          The access tier.
+     * @param callback      The callback that receives the response.
+     */
+    public void setTier(String containerName,
```

Comment:
Should this be setTier or setBlobTier?

Reply:
I just changed mine to be setBlobHttpHeaders. Just cus it's getBlobProperties. Doesn't matter too much though cus archboard.

Reply:
I agree with Rick, we still have to standardize the naming for methods like these, given that we also have things like `upload` and `getBlobProperties` in the same client and both pertain to operations involving a single blob.